types/braintree-web-drop-in/test/braintree-web-drop-in-global.test.ts

```diff
@@ -1,3 +1,5 @@
+import { PaymentMethodRequestablePayload, PaymentOptionSelectedPayload } from "braintree-web-drop-in";
```

Comment:
@saoudrizwan not sure how to avoid using module imports for types in the global version. They are explicitly typed and exported as names so that functions for `.on()`/`.off()` can type their signatures, since `.on()`/`.off()` are reference-based as usual.